src/main/resources/static/css/mapStyle.css

```diff
@@ -0,0 +1,46 @@
+html, body {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.CORRIDOR {
+    background-color: #A9A9A9;
+}
+
+.WALL, .OBSTRUCTION{
+    background-color: rgb(70, 70, 70);
+}
+
+.ROOM {
+    background-color: #BDB76B;
+}
+
+.DOOR {
+    background-color: red;
+}
+
+.tile {
+    color: rgba(0,0,0,0);
+    font-size: 10px;
+    /* border: rgb(0,0,0) solid; */
+    /* border-width: 1px; */
+}
+
+.tile:hover {
+    border: rgb(0,0,0) solid;
+    color: rgba(0,0,0,1);
+    border-width: 1px;
+}
+
+
+.stage-grid{
+	display: grid;
+	grid-template-columns: repeat(30, 1fr);
+	grid-template-rows: repeat(20, 1fr);
+	grid-column-gap: 0;
+	grid-row-gap: 0;
+	height: 60vh;
+	width: 60.0vh;
+}
+
```

Comment:
Jeszcze jedna uwaga do nazewnictwa klas - powinny być one z małej, choć przypuszczam, że wiąże się to z użytym thymeleafem?

Reply:
W htmlu rzeczywiście Thymeleaf jako nazwę klasy pobiera Enuma, którego wartości są pisane CAPSem.
![Image 4](https://user-images.githubusercontent.com/57985744/95028513-fb903c80-06a0-11eb-8387-03e08779f9b5.png)